### PR TITLE
Add support for images to be used as pins

### PR DESCRIPTION
**Pull request by canstudios-nicolaw**
Fixes https://github.com/adaptlearning/adapt_framework/issues/1728

### CODE REVIEW

**Comment by canstudios-nicolaw** [comment]
This was tested for accessibility using Chrome/Firefox/Edge/IE11 and the NVDA screen reader. Please let me know if there are any guidelines as to which browsers etc. accessibility should be tested on.

File: properties.schema [hunk 1]
```diff
@@ -217,18 +217,42 @@
...
+            "required": false,
+            "title": "Item Pin",
+            "properties": {
+              "src": {
+                "type": "string",
+                "required": false,
+                "default": "",
+                "inputType": "Asset:image",
+                "validators": [] ,
+                "help": "The pin image leave blank for default"
```
Comment (cahirodoherty-learningpool) [inline properties.schema]: Minor: I think there should be some punctuation in this help text to have it read better

Comment (canstudios-nicolaw) [reply properties.schema]: :+1:

File: templates/hotgraphic.hbs [hunk 2]
```diff
@@ -17,7 +17,13 @@
         {{/if}}
         {{#each _items}}
           <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
+            {{#if this._pin.src}}
```
Comment (moloko) [inline templates/hotgraphic.hbs]: sorry I know this is quite picky - but there's no need to use `this` here and - to the best of my knowledge - we don't ever include it in any other .hbs files in core adapt. would you mind removing it for consistency?

Comment (danielstorey) [inline templates/hotgraphic.hbs]: Minor: I think `this` can be omitted as the context in `#each` will always be the item `{{#if _pin.src}}` - just makes things easier to read

File: templates/hotgraphic.hbs [hunk 3]
```diff
@@ -17,7 +17,13 @@
...
+              <div class="hotgraphic-graphic-pin-image item-{{@index}}">
+                <img src="{{this._pin.src}}" aria-label="{{this._pin.alt}}">
```
Comment (danielstorey) [inline templates/hotgraphic.hbs]: Minor: as above

File: templates/hotgraphic.hbs (hunk 2 above)
Comment (canstudios-nicolaw) [reply templates/hotgraphic.hbs]: No need to apologise! I'd rather you be picky

Comment (canstudios-nicolaw) [reply templates/hotgraphic.hbs]: Thanks :)

**Comment by canstudios-nicolaw** [comment]
@moloko I have updated the README.md and example.json, good spot.

**Pull request merged**